webapp/home/templates/home/snippets/footer.html

```diff
@@ -78,14 +42,15 @@ <h6>SITE LINKS</h6>
           <div class="col-md-6">
             <small>
               <a href="/request">Support</a> <br>
+              <a href="/about">About</a>a> <br>
```

Comment:
Typo in closing tag

Reply:
Fixed, thanks! That damn autocompletion of tags gets me so often!